docs-ref-conceptual/azure-cli-extension-alias.md

```diff
@@ -49,27 +49,23 @@ The alias extension is under active development and new versions are released re
 az extension update --name alias
 ```
 
-## Alias commands file format
+## Manager Azure CLI aliases
```

Comment:
The first word of `H2` sections in our documentation should generally be an imperative verb, so the reader sees it as a call to action. This H2 should read something like `Manage aliases for the Azure CLI`.